api/src/controllers/contact.ts

```diff
@@ -45,3 +47,46 @@ export const createContact = async (
     throw errorWithCode(message, 500);
   }
 };
+
+/**
+ * POST /requests
+ * @param {Express Request} req
+ * @param {Express Response} res
+ * @returns undefined
+ */
+export const inviteToOrg = async (
+  { params, body }: { params: any, body: any }, res: Response
+): Promise<void> => {
+  logger.info('createInvitationRequest')
+
+  const { githubId: recipient, organizations } = body
+
+  try {
+    logger.info(`user  approved request created for ${recipient}`)
```

Comment:
This is fragmented, if I were to guess it should read:
```suggestion
    logger.info(`user approved, request created for ${recipient}`)
```

If that isn't correct, please update to make it clearer.